src/assets/stylesheets/styles.scss

```diff
@@ -111,34 +111,12 @@
 // NavGlobal
 #nav-global {
   background: color('blue-60') linear-gradient(0deg, rgba(0, 0, 0, 0) 25%, rgba(0, 0, 0, 0.2) 100%);
-  color: #FFFFFF;
-
-  .nav-global-inner {
-    align-items: center;
-    display: flex;
-    @include u-maxw('desktop-lg');
-    @include u-padding-x(2);
-    @include u-margin-x('auto');
-    @include u-font('body', 'xs');
-    @include at-media('desktop') {
-      @include u-padding-x(4);
-    }
 
-    a {
-      align-items: center;
-      display: flex;
-      gap: 0.5rem;
-      padding: units(1.5) 0 units(1);
-      border-width: 0 0 units(0.5) 0;
-      @include u-border('transparent');
-      @include u-text('white');
-      @include at-media('desktop') {
-        padding: units(2.5) 0 units(2);
-      }
-    }
-    
-    a.nav-product {
-      @include u-text('no-underline');
+  a {
+    color: white; // sets login / logout link color
```

Comment:
[nitpick] could we use the `text-white` utility class in lieu of this?

Reply:
Yes, although I had some trouble figuring out how to effectively get it into the Login / Logout button components without using CSS.  I can add it directly to those components, though, if we don't mind them having the inline utilities as well!